assets/enx-redirect/report/header.html

```diff
@@ -0,0 +1,77 @@
+<link rel="shortcut icon" href="/static/favicon.ico">
+<meta name="msapplication-TileColor" content="#ff0554">
+<meta name="msapplication-config" content="/static/browserconfig.xml">
+<meta name="theme-color" content="#ffffff">
+{{.csrfMeta}}
+
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css"
+  integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l" crossorigin="anonymous">
+<link rel="stylesheet"
+  href="https://cdnjs.cloudflare.com/ajax/libs/open-iconic/1.1.1/font/css/open-iconic-bootstrap.min.css"
+  integrity="sha384-wWci3BOzr88l+HNsAtr3+e5bk9qh5KfjU6gl/rbzfTYdsAVHBEbxB33veLYmFg/a" crossorigin="anonymous">
+<link rel="stylesheet"
+  href="https://cdnjs.cloudflare.com/ajax/libs/intl-tel-input/17.0.12/css/intlTelInput.css"
+  integrity="sha384-TXDx4BvGGuJDYIKlcgXfDntJ100A809RRLB4W72MhjXJzakfj3ptxy4zER5qsxZH" crossorigin="anonymous">
+<link rel="stylesheet"
+  href="/static/css/application.css" crossorigin="anonymous" />
+
+<script src="https://code.jquery.com/jquery-3.5.1.min.js"
+  integrity="sha384-ZvpUoO/+PpLXR1lu4jmpXWu80pZlYUAfxl5NsBMWOEPSjUn/6Z/hRTt8+pR6L4N2" crossorigin="anonymous"></script>
+<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js"
+  integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/js/bootstrap.min.js"
+  integrity="sha384-+YQ4JLhjyBLPDQt//I+STsc9iw4uQqACwlvpslubQzn4u2UU2UFM80nGisd026JF" crossorigin="anonymous"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/intl-tel-input/17.0.12/js/intlTelInput.js"
+  integrity="sha384-qqHEed+7J6B7s1PDZUg0ockDBmE8DsXphZqsNwV2pzkH5kXcNP7ZKRYLxbd5f/WL" crossorigin="anonymous"></script>
+<script src="/static/js/application.js?{{.buildID}}" crossorigin="anonymous"></script>
+
+<title>Verification Code Request</title>
+
+<header class="mb-3">
+  <nav class="nav nav-tabs navbar-expand-md navbar-light bg-light">
+    <div class="container">
+      <a class="navbar-brand" href="#">Request Exposure Notification Verification Code</a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+    </div>
+  </nav>
+</header>
+
+{{end}}
+
+{{define "errorable"}}
+{{if .}}
+<div class="invalid-feedback">
+  {{joinStrings . ", "}}
+</div>
+{{end}}
+{{end}}
+
+{{define "errorSummary"}}
+{{if $errs := .Errors}}
+  <div class="alert alert-danger mb-4" role="alert">
+    <p>
+      The following errors occurred:
+    </p>
+
+    <ul class="mb-1">
+      {{range $k, $v := $errs}}
+        {{range $e := $v}}
+          <li><strong>{{$k}}</strong> {{$e}}</li>
+        {{end}}
+      {{end}}
+    </ul>
+  </div>
+{{end}}
+{{end}}
```

Comment:
There's a bunch of files with missing newlines EOF. I'm only commenting once, but please fix them all up (they can make the template renderer sad)